release/docker/opendistroforelasticsearch-data-prepper-0.7.1-alpha.Dockerfile

```diff
@@ -1,6 +1,4 @@
-# building on CentOS 7 to be in line with
-# https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch
-
+# For the v0.7.0-alpha release we will use the amazoncorretto amazon linux.
 FROM amazoncorretto:15-al2-full
 ARG CONFIG_FILEPATH
 ARG JAR_FILE
```

Comment:
should we update this `CMD java $JAVA_OPTS -jar data-prepper.jar ${ENV_CONFIG_FILEPATH}` to also include the configuration file for DataPrepperServer ?

Reply:
Probably not, 0.7.x didn't have server config files at the time